scripts/generated/app.ts

```diff
@@ -1226,6 +1226,7 @@ export const APP_STRING_KEYS = {
     TITLE_FAILED_TO_CONNECT: 'APP.SIGN_INTO_MOBILE.TITLE_FAILED_TO_CONNECT',
     TITLE_SCAN_FROM_APP: 'APP.SIGN_INTO_MOBILE.TITLE_SCAN_FROM_APP',
     TITLE_SIGN_INTO_MOBILE: 'APP.SIGN_INTO_MOBILE.TITLE_SIGN_INTO_MOBILE',
+    VERIFICATION_CODE: 'APP.SIGN_INTO_MOBILE.VERIFICATION_CODE',
```

Comment:
looks like this was missing from a prev pr; cc @mike-dydx in case you've been trying to use it